<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emoji Guessing Game</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Emoji Guessing Game</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="emoji_display">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>700</width>
      <height>50</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="hint_display">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>700</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Hint: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>

<widget class="QLineEdit" name="answer_input">
    <property name="geometry">
        <rect>
            <x>200</x>
            <y>210</y>
            <width>400</width>
            <height>40</height>
        </rect>
    </property>
    <property name="placeholderText">
        <string>Enter your guess here...</string>
    </property>
    <property name="alignment">
        <set>Qt::AlignCenter</set> <!-- Center align the text -->
    </property>
    <property name="styleSheet">
        <string>
            font-size: 18px;  /* Increase font size */
            color: grey;      /* Color for placeholder text */
        </string>
    </property>
</widget>


   <widget class="QPushButton" name="btn_check">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Answer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_next">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show_answer">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>270</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Answer</string>
    </property>
   </widget>
   <widget class="QLabel" name="score_display">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Score: 0</string>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="attempts_display">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Attempts Left: 3</string>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
